### PR TITLE
Fix stopping QEMUs

### PR DESCRIPTION
**Pull request by ivan4th**
Get rid of sudo-nsenter-sudo construct

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/436)

### CODE REVIEW

File: build/cmd.sh [hunk 1]
```diff
@@ -407,6 +407,9 @@ case "${cmd}" in
     integration)
         ( vcmd 'build/cmd.sh run-integration-internal' )
         ;;
+    install-vendor-internal)
+        ( vcmd "build/cmd.sh install-vendor-internal" )
```
Comment (jellonek) [inline build/cmd.sh]: Are you sure about that? Looks like endless loop...

**Comment by jellonek** [comment]
Reviewed 6 of 7 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[cmd/vmwrapper/vmwrapper.go, line 84 at r3](https://reviewable.io:443/reviews/mirantis/virtlet/436#-Kv6Tsh3fWcbjhJK9hqx:-Kv6Tsh3fWcbjhJK9hqy:b-8sfqxl) ([raw file](https://github.com/mirantis/virtlet/blob/a5f9a344c8b533aef8d79e0fb41c151be0fbb199/cmd/vmwrapper/vmwrapper.go#L84)):*
> ```Go
> 
> // This function is a high-priority constructor that will be invoked
> // before any Go code starts.
> ```

Really hacky!
I like that :)

---

*[examples/cirros-vm.yaml, line 41 at r3](https://reviewable.io:443/reviews/mirantis/virtlet/436#-Kv6U5dcub4zPiS-pJdt:-Kv6U5dcub4zPiS-pJdu:b-qb45x9) ([raw file](https://github.com/mirantis/virtlet/blob/e39b61a4eebef051715d60dffab9d16bc0a8e4ba/examples/cirros-vm.yaml#L41)):*
> ```YAML
>     # virtlet/ prefix is used by CRI proxy, the remaining part
>     # of the image name is prepended with https:// and used to download the image
>     image: virtlet/cirros
> ```

Last thing to fix.
Please add also above: `imagePullPolicy: IfNotPresent`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/436)*

**Comment by ivan4th** [comment]
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[build/cmd.sh, line 411 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/436#-Kv52xX7D-JamqtJpns1-r1-411:-Kv6S_aumEX-jh-SixYd:btgpz6r) ([raw file](https://github.com/mirantis/virtlet/blob/a496f80cc746ebc9f9c3f2342fc3e70c44278bed/build/cmd.sh#L411)):*
<details><summary><i>Previously, jellonek (Piotr Skamruk) wrote…</i></summary><blockquote>

Are you sure about that? Looks like endless loop...
</blockquote></details>

Fixed, already noticed that, thanks

---

*[examples/cirros-vm.yaml, line 41 at r3](https://reviewable.io:443/reviews/mirantis/virtlet/436#-Kv6U5dcub4zPiS-pJdt:-Kv6VQKa6QNinnLx20Y0:b4m6xbe) ([raw file](https://github.com/mirantis/virtlet/blob/e39b61a4eebef051715d60dffab9d16bc0a8e4ba/examples/cirros-vm.yaml#L41)):*
<details><summary><i>Previously, jellonek (Piotr Skamruk) wrote…</i></summary><blockquote>

Last thing to fix.
Please add also above: `imagePullPolicy: IfNotPresent`

</blockquote></details>

well, let's do this in a separate PR perhaps, as the purpose of this one is a bit different.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/436)*

**Comment by jellonek** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[examples/cirros-vm.yaml, line 41 at r3](https://reviewable.io:443/reviews/mirantis/virtlet/436#-Kv6U5dcub4zPiS-pJdt:-Kv6jZvGzu_hius8SFlU:b8jeto2) ([raw file](https://github.com/mirantis/virtlet/blob/e39b61a4eebef051715d60dffab9d16bc0a8e4ba/examples/cirros-vm.yaml#L41)):*
<details><summary><i>Previously, ivan4th (Ivan Shvedunov) wrote…</i></summary><blockquote>

well, let's do this in a separate PR perhaps, as the purpose of this one is a bit different.
</blockquote></details>

Good idea. Will prepare such PR today.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/436#-:-Kv6jcyEhnqiQ4E9jLmZ:bnfp4nl)*

**Comment by pigmej** [comment]
:lgtm: but let's wait for @loles 

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/436#-:-Kv6ta-NWyXjkGOi1013:b-t90s42)*

**Pull request merged**